elide-graphql/src/main/java/com/yahoo/elide/graphql/ModelBuilder.java

```diff
@@ -69,12 +69,14 @@
 
     /**
      * Class constructor, constructs the custom arguments to handle mutations
-     * @param dictionary elide entity dictionary
+     * @param entityDictionary elide entity dictionary
      * @param dataFetcher graphQL data fetcher
```

Comment:
we can add nonEntityDictionary in the param list to reduce warnings. :)

Reply:
done